.github/workflows/generate-base-python.yml

```diff
@@ -1,9 +1,9 @@
 name: generate-base-python
 on:
   schedule:
-    # run 15 minutes after midnight (UTC) weekly on sundays
+    # run at noon on sundays to prepare for monday
     # used https://crontab.guru/ to generate
-    - cron: '15 0 * * SUN'
+    - cron: '0 12 * * SUN'
```

Comment:
curious why change the cron schedule?

Reply:
Because, I noticed it wasn't actually working and CI was still getting triggered on the first Monday morning PR. So, rather than try to be cute keeping it close to midnight I just moved it to the middle of the day.